cmd/fleetctl/get.go

```diff
@@ -324,14 +324,39 @@ func getQueriesCommand() *cli.Command {
 					return fmt.Errorf("could not list queries: %w", err)
 				}
 
+				me, err := client.Me()
+				if err != nil {
+					return err
+				}
+				ok, err := userIsObserver(me)
+				if err != nil {
+					return err
+				}
+				if ok {
+					// Filter out queries (in-place) that a observer user
+					// cannot execute (this behavior matches the UI).
+					n := 0
+					for _, query := range queries {
+						if query.ObserverCanRun {
+							queries[n] = query
+							n++
+						}
+					}
+					queries = queries[:n]
```

Comment:
Don't know if this is possible, but if `queries == nil` this will blow up

Reply:
I believe we are covered: https://go.dev/play/p/gFaXjSKcqgs